lib/money/money.rb

```diff
@@ -6,6 +6,10 @@ class Money
 
   attr_reader :value, :cents
 
+  def self.from_amount(value)
+    new(value)
+  end
```

Comment:
That'll work because offiste_payment has this line just for us: https://github.com/activemerchant/offsite_payments/blob/e90298b1801552070ca9cee83f649363567cb8a9/lib/offsite_payments/notification.rb#L37

It's not a perf hotspot AFAIK but a conditional checking for [the method arity](https://ruby-doc.org/core-2.3.0/Method.html#method-i-arity) would be cleaner there.